src/group-constructors.js

```diff
@@ -55,6 +55,8 @@
     var underlyingPlayer;
     var ticker = function(tf) {
       var player = underlyingPlayer._wrapper;
+      if (player.playState == 'pending') return;
```

Comment:
I think we need to determine whether the specification should change to permit us calling again first. Let's commit this for now and work out the specification question later.